curator-recipes/src/main/java/org/apache/curator/framework/recipes/leader/LeaderLatch.java

```diff
@@ -190,6 +190,12 @@ public void close() throws IOException
         close(closeMode);
     }
 
+    // for testing
```

Comment:
```suggestion
    @VisibleForTesting
```
Isn't this annotation the better way of documenting this here?

Reply:
For me, `VisibleForTesting` can mean that this method can be used in production but its visibility is wider than necessary for testing.

But it's fair enough to use `VisibleForTesting`. I don't want to make the change now :)